barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_integration.test.cpp

```diff
@@ -543,4 +543,26 @@ TEST_F(AcirIntegrationTest, DISABLED_UpdateAcirCircuit)
     EXPECT_TRUE(prove_and_verify_honk<Flavor>(circuit));
 }
 
+/**
+ * @brief Test recursive honk recursive verification
+ *
+ */
+TEST_F(AcirIntegrationTest, DISABLED_HonkRecursion)
+{
+    using Flavor = UltraFlavor;
+    using Builder = Flavor::CircuitBuilder;
+
+    std::string test_name = "verify_honk_proof"; // arbitrary program with RAM gates
+    auto acir_program = get_program_data_from_test_file(
+        test_name,
+        /*honk_recursion=*/false); // WORKTODO: TODO(https://github.com/AztecProtocol/barretenberg/issues/1013):
```

Comment:
why is this false here?

Reply:
Its false because with this new pattern, the `honk_recursion` flag is only used for the "this circuit will be recursively verified with honk" use case. The verifier is set to a honk verifier via the noir `verify_proof_with_type` call directly

Reply:
added a comment to this effect